test-support/helpers/ember-tooltips.js

```diff
@@ -105,37 +184,39 @@ export function assertTooltipVisible(assert, options={}) {
 export function assertTooltipSide(assert, options = {}) {
   const { side } = options;
 
-  const sideIsValid = (
-    side === 'top' ||
-    side === 'right' ||
-    side === 'bottom' ||
-    side === 'left'
-  );
+  validateSide(side);
 
-  /* We make sure the side being tested is valid. We
-  use Ember.assert because assert is passed in from QUnit */
+  const { expectedGreaterDistance, expectedLesserDistance } = getPositionDifferences(options);
 
-  if (!sideIsValid) {
-    Ember.assert(`You must pass side like assertTooltipSide(assert, { side: 'top' }); Valid options for side are top, right, bottom, and left.`);
-  }
+  /* When the side is top or left, the greater number
+  is the target's position. Thus, we check that the
+  target's position is greater than the tooltip's
+  position. */
 
-  const expectedSide = options.side || 'top';
-  const $target = getTooltipTargetFromBody(options.targetSelector);
-  const targetPosition = $target[0].getBoundingClientRect();
-  const $tooltip = getTooltipFromBody(options.selector);
-  const tooltipPosition = $tooltip[0].getBoundingClientRect();
+  assert.ok(expectedGreaterDistance > expectedLesserDistance,
+    `Tooltip should be on the ${side} side of the target`);
+}
+
+export function assertTooltipSpacing(assert, options) {
```

Comment:
I thought about making `assertTooltipSide()` and `assertTooltipSpacing()` a single helper, given that `assertTooltipSpacing()` requires a `side` but decided to keep one assertion per option. Happy to reconsider this.